web/modules/custom/farm_nfa/src/Form/ForestEditPlanPermissionForm.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Drupal\farm_nfa\Form;
+
+use Drupal\Core\Entity\ContentEntityForm;
+
+/**
+ * Forest plan permissions edit form.
+ *
+ * @ingroup farm_nfa
+ */
+class ForestEditPlanPermissionForm extends ContentEntityForm {
```

Comment:
Is there any reason you need to create this form? I think you could set the callback to ContentEntityForm directly

Reply:
yes @pcambra, i added this here i can we needed to add any custom validations here !